opencti-platform/opencti-front/src/private/components/settings/users/UserHistory.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { FunctionComponent, useEffect, useState } from 'react';
+import Typography from '@mui/material/Typography';
+import { useQueryLoader } from 'react-relay';
+import { LogsOrdering, OrderingMode, UserHistoryLinesQuery } from '@components/settings/users/__generated__/UserHistoryLinesQuery.graphql';
+import Tooltip from '@mui/material/Tooltip';
+import IconButton from '@mui/material/IconButton';
+import VisibilityOutlinedIcon from '@mui/icons-material/VisibilityOutlined';
+import PolylineOutlinedIcon from '@mui/icons-material/PolylineOutlined';
+import { Link } from 'react-router-dom';
+import makeStyles from '@mui/styles/makeStyles';
+import { GqlFilterGroup } from '../../../../utils/filters/filtersUtils';
+import { useFormatter } from '../../../../components/i18n';
+import Loader, { LoaderVariant } from '../../../../components/Loader';
+import SearchInput from '../../../../components/SearchInput';
+import UserHistoryLines, { userHistoryLinesQuery } from './UserHistoryLines';
+
+const useStyles = makeStyles(() => ({
+  createdByUserRedirectButton: {
+    float: 'left',
+    marginTop: -15,
+  },
+}));
```

Comment:
```suggestion
const createdByUserRedirectButton = {
  float: 'left',
  marginTop: -15,
};
```

And use this object directly in the div, to avoid calling deprecated makeStyle